src/content/docs/ko/reference/errors/incorrect-strategy-for-i18n.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: You can't use the current function with the current strategy
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **IncorrectStrategyForI18n**: The function `FUNCTION_NAME\' can only be used when the `i18n.routing.strategy` is set to `"manual"`.
```

Comment:
```suggestion
> **IncorrectStrategyForI18n**: The function `FUNCTION_NAME` can only be used when the `i18n.routing.strategy` is set to `"manual"`.
```

fixed this in https://github.com/withastro/docs/pull/7876

Reply:
thank you!